docs/user-guide/common-use-cases/how-to-make-a-transaction.md

```diff
@@ -1,47 +1,109 @@
 
 Which returns a list of all transactions for this particular wallet. Optional range filters can be provided. A transaction will go through a succession of states, starting as “Pending”. If a transaction stays pending for too long (because rejected by a mempool, or because lost in translation due to multiple chain switches), users may decide to forget it using:
 
-[`DELETE /v2/byron-wallets/{walletId}/transactions/{transactionId}`](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/deleteByronTransaction)
+[`DELETE /wallets/{walletId}/transactions/{transactionId}`](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/deleteTransaction)  - transactions from **Shelley** wallet
+[`DELETE /byron-wallets/{walletId}/transactions/{transactionId}`](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/deleteByronTransaction)  - transactions from **Byron** wallet
 
 For more information about transactions lifecycle, have a look at [[About-Transactions-Lifecycle]].
 
-## Signed and serialized transactions 
-::: {.highlight-block}
-**Difficulty:** advanced
 
-**Requires:**
-- 📦 cardano-transactions >= `1.0.0`
-- 📦 cardano-submit-api >= `2.0.0` OR cardano-wallet >= `v2020-04-01`
-:::
 
-Alternatively, `cardano-wallet` and `cardano-submit-api` allows clients to submit already signed and serialized transactions as a raw bytes blob. This can be done by submitting such serialized data as an `application/octet-stream` to either of:
+## Signed and serialized transactions
 
-- cardano-wallet: [`POST   /v2/proxy/transactions`](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/postExternalTransaction)
-- cardano-submit-api: [`POST    /api/submit/tx`](https://input-output-hk.github.io/cardano-rest/submit-api/#operation/postTransaction)
+Alternatively, `cardano-wallet` allows clients to submit already signed and serialized transactions as a raw bytes blob. This can be done by submitting such serialized data as an `application/octet-stream` :
 
-In this scenario, the server engine will verify that the transaction is structurally well-formed and forward it to its associated node. If the transaction belongs to a known wallet, it will eventually show up in the wallet your wallet.
-
-Such transactions can be constructed from raw data using either [cardano-transactions library or command-line interface](https://github.com/input-output-hk/cardano-transactions). Examples and documentation excerpts are available on the corresponding Github repository.
+[`POST /proxy/transactions`](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/postExternalTransaction)
 
+In this scenario, the server engine will verify that the transaction is structurally well-formed and forward it to its associated node. If the transaction belongs to a known wallet, it will eventually show up in the wallet your wallet.
```

Comment:
```suggestion
In this scenario, the server engine will verify that the transaction is structurally well-formed and forward it to the node instance associated with it. If the transaction belongs to a known wallet, it will eventually show up in the wallet.
```